Extensions/EdFi.Ods.Extensions.TPDM/EdFi.Ods.Extensions.TPDM.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
```

Comment:
Same comment here about updating `Autofac.Extensions.DependencyInjection` to `7.2.0` and `Newtonsoft.Json` to `13.0.1`

Reply:
updated